app/assets/stylesheets/application.css.scss

```diff
@@ -233,6 +241,7 @@ body.modal-open {
 
 #repo_list > div.row > div > div > a > img{
   height: 160px;
+  background-color: #F5F5F5;
```

Comment:
Color literals like `#F5F5F5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F5F5F5` should be written as `#f5f5f5`